src/directwrite/mod.rs

```diff
@@ -97,15 +96,12 @@ impl DirectWriteRasterizer {
         self.fonts.get(&font_key).ok_or(Error::FontNotLoaded)
     }
 
-    fn get_glyph_index(&self, face: &FontFace, c: char) -> Result<u16, Error> {
-        let idx = *face
-            .get_glyph_indices(&[c as u32])
-            .first()
+    fn get_glyph_index(&self, face: &FontFace, c: char) -> Option<u16> {
```

Comment:
Another instance of inconsistent `c` if your goal was to make that more consistent across backends.